### ISSUE TITLE
Setting MSRV for cargo workspaces

### ISSUE THREAD

**Issue by orhun** [foresterre/cargo-msrv#316]
Hello!

I have a [cargo workspace](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html) project and it has a `Cargo.toml` as following:

```toml
[workspace]
members = [
    "project1",
    "project2",
]
```

When I run `cargo msrv set 1.54.0`, `Cargo.toml` is updated like this:

```toml
package = { metadata = { msrv = "1.54.0" } }
[workspace]
members = [
    "project1",
    "project2",
]
```

I don't think this is a correct behavior since `cargo run` fails to parse the manifest:

```
error: failed to parse manifest at `Cargo.toml`

Caused by:
  missing field `name` for key `package`
```

I think there are 2 things that could be done here:
1- Parse `Cargo.toml` and fail gracefully if it only contains `[workspace]` section rather than `[package]`
2- Update MSRV's for the all members of the workspace (which might not be ideal for some cases)

What do you think? I'm happy to propose a PR about this.

**Comment by foresterre** [comment]
Thanks for the report! I see I also need to work on the preferred formatting (e.g. `[package.metadata] msrv = "1.54.0"` over `package = { metadata = { msrv = "1.54.0" } }`) :).

I agree that this isn't the correct behavior. Probably we can apply both of your suggestions; the first one by default, and the second one when a flag (e.g. `--all-workspace-crates`) is provided.

An alternative could be to define the msrv in the [workspaces.metadata](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-workspacemetadata-table) table; but this has some disadvantages:
* Not standardized (similar to any Cargo version which does not support `rust-version`, where cargo-msrv outputs to the `package.metadata.msrv` field like in your example)
* Requires cargo-msrv (and other tooling) to also support it as an input, otherwise it has only value to human readers

I think it would be best to start with your first suggestion, and potentially add an option for the second.

**Comment by foresterre** [comment]
Fixed by #317, thanks @orhun!
